react-native/android/analytics.gradle

```diff
@@ -57,8 +57,9 @@ class SendAnalyticsTask extends DefaultTask {
     def sendAnalytics() {
        try {
             def env = System.getenv()
-            def disableAnalytics= env['REALM_DISABLE_ANALYTICS']
-            if (disableAnalytics == null || disableAnalytics != "true") {
+            def disableAnalytics = env['REALM_DISABLE_ANALYTICS'] != null
+            def isCI = env['CI'] != null
```

Comment:
```suggestion
            def isCI = env['CI'] != null || env['CI'] == "true"
```

Reply:
Similarly with `REALM_DISABLE_ANALYTICS`, I think it makes sense to check for existence rather than a particular value. I don't suppose many developers have a `CI` environment variable set on their machines.